DESIGN.md

```
@@ -0,0 +1,505 @@
+| `success` | #16a34a | 성공 메시지, 완료 상태 |
+| `warning` | #ca8a04 | 경고 |
+| `error` | #dc2626 | 에러, 삭제 |
+| `info` | #18181b | 정보 알림 (= primary) |
+
+각 시맨틱 색상은 `-muted` 변형(연한 배경용)을 함께 제공합니다.
+
+## Typography
+
+### Font Family
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)